largest_num.js

```diff
@@ -1,2 +1,25 @@
 alert('hello world');
-prompt('Soke');
\ No newline at end of file
+prompt('Soke');
+
+//Exercise(5) number one(1)
+//Write a function that takes an array of numbers and returns the largest number in the array.
+
+// This is the Pseudocode
+//- we set a function of largestArray to it's prim array.
+//- we set a variable of largestNumber to the first index of the array.
+//- we set a FOR loop by initializing index to 1 and set a condition of the length of am array greater than index and increment index.
+//- we set am IF statement of index of array greater than largestNumber, and equalize both.
+//- we return largestNumber.
+
```

Comment:
Kindly do ont start you Pseudocode by mentioning the function.
please look the example below:

- [ ] Define an array of numbers
- [ ] Initialize a variable to hold the largest number
- [ ] Loop through the array and compare each number to the largest number
- [ ] Finally return the result